plugins/inputs/jolokia/jolokia.go

```diff
@@ -65,6 +73,20 @@ const sampleConfig = `
   #   host = "127.0.0.1"
   #   port = "8080"
 
+  ## Optional http timeouts
+  ##
+  ## ResponseHeaderTimeout, if non-zero, specifies the amount of
+  ## time to wait for a server's response headers after fully
```

Comment:
this is too much writing, decrease the verbosity of this to match the rabbitmq plugin (you'll need to update master to see it)